### PR TITLE
iframe integration v1

### PR DESCRIPTION
**Pull request by Connor-Bernard**
iFrame integration with css, yml, html, py file

### CODE REVIEW

**Comment by dahluwalia** [comment]
iFrame integration with css, yml, html, py file

File: website/src/views/conceptMap.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 
 export default function ConceptMap() {
+
     return (
-        <div>
+        <div style={{ textAlign: 'center' }}>
             <h1>Concept Map</h1>
+            <iframe className="concept_map_iframe" id="ConceptMap" title="Concept Map"
+                src="http://localhost:8080?show_legend=false"
```
Comment (Connor-Bernard) [inline website/src/views/conceptMap.js:9]: This should be a container name

File: website/src/css/app.css
Comment (Connor-Bernard) [inline website/src/css/app.css:1]: Can you export this into its own file for the change

File: website/src/views/conceptMap.js (hunk 1 above)
Comment (Connor-Bernard) [reply website/src/views/conceptMap.js]: Resolved in [c67e6c0](https://github.com/Connor-Bernard/gradeView/pull/96/commits/c67e6c0ec1c1415fbd8a9084e7c270b322e31045)

File: website/src/css/app.css
Comment (Connor-Bernard) [reply website/src/css/app.css:1]: Resolved in [8d221d9](https://github.com/Connor-Bernard/gradeView/pull/96/commits/8d221d9f9fbd3236b0ef51cf7f3cbdd0b7745432).

**Pull request merged**